plugins/chat/assets/javascripts/discourse/controllers/chat-message-from-params.js

```diff
@@ -0,0 +1,8 @@
+import Controller from "@ember/controller";
+import { inject as service } from "@ember/service";
+
+export default class ChatMessageFromParamsController extends Controller {
+  @service chat;
```

Comment:
is this needed?

Reply:
I think we can safely remove the chat service, I was testing something with this and forgot to remove it.